src/pint/observatory/special_locations.py

```diff
@@ -315,8 +330,16 @@ def posvel(self, t, ephem, group=None):
         return geo_posvel + stl_posvel
 
 
-# Need to initialize one of each so that it gets added to the list
-BarycenterObs("barycenter", aliases=["@", "ssb", "bary", "bat"])
-GeocenterObs("geocenter", aliases=["0", "o", "coe", "geo"])
-T2SpacecraftObs("stl_geo", aliases=["STL_GEO"])
-# TODO -- BIPM issue
+def load_special_locations():
```

Comment:
Is it safe to rerun this? Like if nothing's been removed can you safely rerun this anyway? I assume so but it'd be good to say so in the docs, since the topo_obs version has the `overwrite` flag.

Reply:
I didn't put that in because the special locations (so far) never change, so if they are loaded they shouldn't need to be reloaded.  But it would throw an exception, so I put in `overwrite=True`